test/experimental/test_with_asset.py

```diff
@@ -210,20 +219,16 @@ def test_builtin_pretrained_sentencepiece_processor(self):
 
     # we separate out these errors because Windows runs into seg faults when propagating
     # exceptions from C++ using pybind11
-    @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
+    # @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
```

Comment:
Please remove whole lines instead of commenting out, before merging.

Reply:
Yup will do. I just commented out for testing purposes :)